app/src/features/settings/components/BillingTeam/components/BillingDetails/MyBillingTeamDetails/components/AddMembersDrawer/components/InviteMembersForm/InviteMembersForm.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from "react";
+import { useSelector } from "react-redux";
+import { getBillingTeamById } from "store/features/billing/selectors";
+import EmailInputWithDomainBasedSuggestions from "components/common/EmailInputWithDomainBasedSuggestions";
+import { RQButton } from "lib/design-system/components";
+import { addUsersToBillingTeam } from "backend/billing";
+import { PostUserAdditionView } from "./components/PostUserAdditionView/PostUserAdditionView";
+import { ExternalDomainWarningBanner } from "./components/ExternalDomainWarningBanner/ExternalDomainWarningBanner";
+import Logger from "../../../../../../../../../../../../../common/logger";
+import "./inviteMembersForm.scss";
+
+interface InviteMembersFormProps {
+  billingId: string;
+  toggleInviteFormVisibility: () => void;
+  closeAddMembersDrawer: () => void;
+}
+
+export const InviteMembersForm: React.FC<InviteMembersFormProps> = ({
+  billingId,
+  toggleInviteFormVisibility,
+  closeAddMembersDrawer,
+}) => {
+  const billingTeamDetails = useSelector(getBillingTeamById(billingId));
+  const [isPostUserAdditionViewVisible, setIsPostUserAdditionViewVisible] = useState(false);
+  const [externalDomainEmails, setExternalDomainEmails] = useState([]);
+  const [isExternalDomainWarningBannerClosed, setIsExternalDomainWarningBannerClosed] = useState(false);
+  const [nonExisitingEmails, setNonExisitingEmails] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [emails, setEmails] = useState([]);
+
+  const handleAddMembersToBillingTeam = useCallback(() => {
+    setIsLoading(true);
+    addUsersToBillingTeam(billingId, emails)
```

Comment:
If no emails present we will still make backend call, add some validations

https://github.com/requestly/requestly/assets/61556757/a24259e4-f9cc-4dc1-838d-2bc1a454b141